frontend/src/components/Secrets/GSecretDialogDDNS.vue

```diff
@@ -0,0 +1,245 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+<template>
+  <g-secret-dialog
+    v-model="visible"
+    :data="secretData"
+    :secret-validations="v$"
+    :secret="secret"
+    vendor="netlify-dns"
```

Comment:
double check -> copy pasta

Reply:
removed obsolete params in dialogs that are not used for multiple vendor kinds (not infra+dns)